app/konnect/api-products/productize-service.md

```diff
@@ -20,7 +20,7 @@ You can set up an API product and API product version by clicking {% konnect_ico
 
 1. Select **API Product** from the API products dashboard to add a new API product.
 
-1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then press **create**. 
+1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then click **Create**. 
 
```

Comment:
can we format lines 31-40? the curl command is not formatted correctly: https://deploy-preview-7561--kongdocs.netlify.app/konnect/api-products/productize-service/#create-an-api-product